front_end_pocket_poll/lib/groups_widgets/group_page.dart

```diff
@@ -16,14 +16,15 @@ import 'package:front_end_pocket_poll/imports/result_status.dart';
 import 'package:front_end_pocket_poll/imports/users_manager.dart';
 import 'package:front_end_pocket_poll/models/event_card_interface.dart';
 import 'package:front_end_pocket_poll/models/get_group_response.dart';
-import 'package:front_end_pocket_poll/utilities/utilities.dart';
 
 import 'group_settings.dart';
 
 class GroupPage extends StatefulWidget {
   final String groupId;
   final String groupName;
 
+  static final int maxEventBatchesInMemory = 3;
```

Comment:
I think we can probably increase this to like 5 yes?

Reply:
leaving at 3 with the intention to raise the batch size later